app/controllers/application_controller.rb

```diff
@@ -1,5 +1,26 @@
 class ApplicationController < ActionController::Base
-  # Prevent CSRF attacks by raising an exception.
-  # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  def current_user
+    current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
```

Comment:
Usual pattern would be to make this @current_user so that it is remembered across calls to the method.